corehq/apps/linked_domain/templates/linked_domain/tabs/pull_release_content_tab.html

```diff
@@ -0,0 +1,80 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div id="ko-tabs-pull-content" class="ko-template">
+  <div data-bind="if: master_link">
+    <p>
+      {% blocktrans %}
+        This project is linked to the upstream project <a data-bind="attr: {'href': master_href}, text: master_link.master_domain"></a>.<br/>
+        The following items below are synced from the upstream project.<br/>
+        <a href="#" target="_blank">Learn more</a> about Linked Projects.
+      {% endblocktrans %}
+    </p>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">{% trans 'Linked Data Models' %}</h3>
+      </div>
+      <div class="panel-body">
+        <table class="table table-striped table-hover">
+          <thead>
+          <tr>
+            <th class="col-sm-5">{% trans "Linked Model" %}</th>
+            <th class="col-sm-2">{% trans "Last Updated" %} ({{ timezone }})</th>
+            <th class="col-sm-5"></th>
+          </tr>
+          </thead>
+          <tbody data-bind="foreach: model_status">
+          <tr>
+            <td data-bind="text: name"></td>
+            <td data-bind="text: lastUpdate"></td>
+            <td data-bind="css: {'has-error': error}">
+              <button class="btn btn-danger" data-toggle="modal" data-bind="visible: showUpdate() && !update_url, attr: {'data-target': '#overwrite-content-modal-' + $index()}">
+                <i class="fa fa-refresh"></i> {% trans "Overwrite" %}
+              </button>
+              <div class="modal fade" tabindex="-1" role="dialog" data-bind="attr: {'id': 'overwrite-content-modal-' + $index()}">
+                <div class="modal-dialog" role="document">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <button type="button" class="close" data-dismiss="modal" aria-label='{% trans_html_attr "Close" %}'><span aria-hidden="true">&times;</span></button>
+                      <h4 class="modal-title">{% trans 'Overwrite' %}</h4>
+                    </div>
+                    <div class="modal-body">
+                      <h4>
+                        {% blocktrans %}
+                          Are you sure you want to overwrite <b data-bind="text: name"></b>?
+                        {% endblocktrans %}
+                      </h4>
+                      <p>
+                        {% blocktrans %}
+                          This will pull <b data-bind="text: name"></b> from the upstream project <b data-bind="text: $parent.master_link.master_domain"></b>, overwriting the existing <b data-bind="text: name"></b>.<br/>
```

Comment:
I probably shouldn't wrap each instance of the data model in bold text. The text in the mockup seemed to only apply to domain level data models (settings, roles) rather than individual models like reports. This was my attempt at making it applicable to both, but not sure I'm a fan. See the screenshot to get a better sense of what this looks like.

Reply:
yeah i would leave off the bold text here and just use a `ko` pseudo element
```
<!-- ko text: name --><!-- /ko -->
```